src/org/usfirst/frc4904/robot/commands/ShootCycle.java

```diff
@@ -0,0 +1,15 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class ShootCycle extends CommandGroup {
+	public ShootCycle() {
+		super("ShootCycle");
+		requires(RobotMap.Component.shooter);
+		addParallel(new Align());
+		addSequential(new FlywheelSpinup());
+		addSequential(new Shoot());
```

Comment:
I'm no rocket scientist, but I do understand addParallel and the like, and it looks to me like it could shoot before it finishes aligning. Please correct for this if it's a problem, and if it isn't, please explain why not.